netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -49,6 +54,13 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 2.14
+ * @todo #750:30min Solve the puzzle for build mysteriously failing for
+ *  this task citing a NullPointerException (See qulice issue #608 raised for
+ *  the same) and then remove TkStart.java from qulice exceptions.
+ * @todo #750:30min Implement a proper Guava cache with eviction policy instead
```

Comment:
@prahladyeri if you already implemented cache cleanup we don't need this puzzle 
lets's remove `TkStart#clearCache()` call from `TkStart#@act()`
and schedule it  like this
```
        public TkStart(final Base bse) {
            this.base = bse;
            Executors.newSingleThreadScheduledExecutor().scheduleAtFixedRate(
                new Runnable() {
                    @Override
                    public void run() {
                        TkStart.this.clearCache();
                    }
                }, 1L, TimeUnit.MINUTES
            );
        }
```